waspc/src/Wasp/Generator/NpmInstall.hs

```diff
@@ -1,5 +1,6 @@
 module Wasp.Generator.NpmInstall
-  ( installNpmDependenciesWithInstallRecord,
+  ( installProjectNpmDependencies,
```

Comment:
I had to export this and kind of break encapsulation. @Martinsos that cool?

Reply:
Yeah, I guess it is fine! Maybye put it as second in the list of exported stuff.
Also, you can always make the move of creating Wasp.Generator.NpmInstall.Internal and putting it there, if you do want to make it clear it is an internal thing but still allow usage if somebody really knows what they are doing. Maybe you don't have to do this here though.

Reply:
This has been removed since.